open-api/lang-parser/src/main/jjtree/io/deephaven/lang/generated/Chunker.jjt

```diff
@@ -599,10 +826,20 @@ checkInterrupts();
     } ) // optional semi-colon delimiter
    } catch (ParseException e) {
      // Whenever there is an unhandled parse exception,
-     // gobble up the rest of the line and attach to previous node.
+     // gobble up the rest of the line and attach to previous node...
+     // unless the current item on the stack ends w/ a valid statement terminator
      if (expr != null) {
-       token = expr.addJunk(eatJunk());
-       fixState(token);
+       Token tok = expr.jjtGetLastToken();
+//       switch(tok.kind) {
+//           case NEWLINE:
```

Comment:
whoops, my comment about generating code that is commented out actually goes here, with another level of depth to it

Reply:
aye, deleted junk, attached new ticket #s to comments.